apps/connect/src/env/token-bridge.mainnet.ts

```diff
@@ -872,6 +872,30 @@ export const ENV: Env = {
           coinGeckoId: "renzo",
           decimals: 9,
         },
+        Op: {
+          key: "Op",
+          symbol: "OP",
+          nativeChain: "Optimism",
+          tokenId: {
+            chain: "Optimism",
+            address: "0x4200000000000000000000000000000000000042",
+          },
+          icon: "https://assets.coingecko.com/coins/images/25244/standard/Optimism.png?1696524385",
+          coinGeckoId: "optimism",
+          decimals: 18,
```

Comment:
you should include `defaults: 8` to cover injective case

```suggestion
         decimals: {
            default: 8,
            Ethereum: 18,
          },
```